spec/spec_helper.rb

```diff
@@ -60,7 +60,7 @@
     add_group 'Identity', 'modules/identity/'
     add_group 'MebApi', 'modules/meb_api/'
     add_group 'Mobile', 'modules/mobile/'
-    add_group 'Notify', 'modules/notify/'
+    add_group 'VaNotify', 'modules/va_notify/'
```

Comment:
One nitpick: Would you be able to alphabetize this entry? I believe the module generator does the alphabetization (which was originally correct when the module was named 'Notify')